docs/development/workflow/get_devel_version.rst

```diff
@@ -229,6 +229,16 @@ ahead to the more sophisticated method look at :ref:`virtual_envs`).
     + You use Python 3.
     + You want to work on C or Cython code in `Astropy`_.
 
+Now is the time to check that you have all the required dependencies installed on 
+your machine to make sure the pip install would be a successfully one. Namely, as 
+you are effectively building from source, please check that you have both "cython"
+as well as "jinja2" installed using pip. (Note: Using conda to installed these two 
+dependencies would normally not work.) Please note that by pip we really mean pip3. 
```

Comment:
Why would using conda normally not work? Is this true?

Reply:
Yup, I tried conda first, no luck. Then I used pip, then it worked.

Reply:
Could be because I was using the development version though, which was installed using pip3.